docs/api/sql/Raster-operators.md

```diff
@@ -1004,6 +1004,111 @@ Output:
 ```
 4
 ```
+### RS_Resample
+
+Introduction:
+Resamples a raster using a given resampling algorithm and new dimensions (width and height), a new grid corner to pivot the raster at (gridX and gridY) and a set of 
+georeferencing attributes (scaleX and scaleY).
+
+RS_Resample also provides an option to pass a reference raster to draw the georeferencing attributes out of. However, the SRIDs of the input and reference raster must be same, otherwise RS_Resample throws an IllegalArgumentException.
+
+For the purpose of resampling, width-height pair and scaleX-scaleY pair are mutually exclusive, meaning any one of them can be used at a time. 
+
+The `useScale` parameter controls whether to use width-height or scaleX-scaleY. If `useScale` is false, the provided `widthOrScale` and `heightOrScale` values will be floored to integers and considered as width and height respectively (floating point width and height are not allowed). Otherwise, they are considered as scaleX and scaleY respectively.
+
+Currently, RS_Resample does not support skewed rasters, and hence even if a skewed reference raster is provided, its skew values are ignored. If the input raster is skewed, the output raster geometry and interpolation may be incorrect. 
+
+The default algorithm used for resampling is `NearestNeighbor`, and hence if a null, empty or invalid value of algorithm is provided, RS_Resample defaults to using `NearestNeighbor`. However, the algorithm parameter is non-optional.
+
+Following are valid values for the algorithm parameter (Case-insensitive):
+
+1. NearestNeighbor
+2. Bilinear
+3. Bicubic
+
+!!!Tip
+    If you just want to resize or rescale an input raster, you can use RS_Resample(raster: Raster, widthOrScale: Double, heightOrScale: Double, useScale: Boolean, algorithm: String)
+Format: 
+
+`RS_Resample(raster: Raster, widthOrScale: Double, heightOrScale: Double, gridX: Double, gridY: Double, useScale: Boolean, algorithm: String)`
+
+`RS_Resample(raster: Raster, widthOrScale: Double, heightOrScale: Double, useScale: Boolean, algorithm: String)`
+
+`RS_Resample(raster: Raster, referenceRaster: Raster, useScale: Boolean, algorithm: String)`
```

Comment:
Can you please use a code block instead of inline code?

Reply:
Done, thanks